README.md

```diff
@@ -343,11 +406,14 @@ W->>W: Close
 
 ## Credential Issuance
 
+Credential Issuance is an [OID4VCI Pre-Authorized Code flow](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#name-pre-authorized-code-flow). 
```

Comment:
At some point we have to talk about the auth flow for repeat customers, right? I presume our KCC schema will require an expiration date. Perhaps this is fine for now.

Reply:
Mmhmm. Reissuing isn't clear to me. There was talk in Oakland of simply revoking any credential that becomes out of date, and issuing a wholly new one. Seem likely that we'll need user interaction for data refresh anyway. OID4VCI calls this out:

> Second, the Credential Issuer can reissue the Credential by starting the issuance process from the beginning. This would involve interaction with the End-User. A Credential needs to be reissued if the Wallet does not have a valid Access Token or a valid Refresh Token. With this approach, when a new Credential is issued, the Wallet might need to check if it already has a Credential of the same type and, if necessary, delete the old Credential. Otherwise, the Wallet might end up with more than one Credential of the same type, without knowing which one is the latest.

Reply:
Should re-issuing always require users to resubmit PII?

Reply:
In some cases yes in others no. For example, if we collected an mDL that expired we'd probably want them to represent